config/printer-elegoo-neptune3-pro-2023.cfg

```diff
@@ -0,0 +1,135 @@
+# Config for Elegoo Neptune 3 Pro (ZNP Robin Nano_DW V2.2)
+#
+# In 'make menuconfig' choose the following options:
+#
+# * Microcontroller architecture: STMElectronics STM32
+# * Processor model: STM32F401
+# * Bootloader offset: 32KiB bootloader
+# * Communication interface: Serial (on USART1 PA10/PA9)
+#
+# `make flash` will not work for this board. Instead, do the following:
+#
+# * Rename `out/klipper.bin` to `out/ZNP_ROBIN_NANO.bin`
+# * Copy `out/ZNP_ROBIN_NANO.bin` to the root of an SD card formatted to FAT32
+# * Insert the card, then restart
+#
+# The display, if connected, will loop in the 'Updating firmware' screen. This
+# is normal, and the flash itself will only take a few seconds.
+#
+# See docs/Config_Reference.md for a description of parameters.
+
+# Core
+
```

Comment:
looks fine in terms of the config, but your intro section does not conform to the prescribed format, according to point 7 i in https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md. I would suggest:
```
# This file contains pin mappings for the stock Elegoo Neptune 3 Pro (ZNP Robin Nano_DW V2.2)
# To use this config, during "make menuconfig" select the
# STM32F401 with a "32KiB bootloader" and serial (on USART1 PA10/PA9)
# communication.

# Note that the "make flash" command does not work with MKS Robin
# boards. After running "make", rename the out/klipper.bin file to out/ZNP_ROBIN_NANO.bin
# Copy the file out/ZNP_ROBIN_NANO.bin to an SD card formatted to FAT32
# and then restart the printer with the SD card inserted.

# See docs/Config_Reference.md for a description of parameters.```

I assume this board does not the usual robin script running on it to create a useable bin file? ie `./scripts/update_mks_robin.py`

Apart from that, all looks good.

Thanks
James